bring-it-up/bring-it-up
# Update/update schema and data

## Description
#### Summary:
This PR updates the data model to improve fields/attributes and updates the data according to new Mongoose schema.

#### Changes:
- Change serviceType, delivery, and urgency to enums
- Make location and school nullable
- Add (nullable) logo field
- Replace old data in counselling-services.json with more uniform data according to the updated schema

#### How to test this PR:
- Can be tested by adding a new service to the database using the new schema:
1. Connect to the test database "test_db" by changing the database name in the .env file. The connection string should look like: mongodb+srv://<USER>:<PASSWORD>@cluster0.goe5y.mongodb.net/<DATABASE_NAME>?retryWrites=true&w=majority.
2. Use the POST request to add a service to the database. You can either create your own request for this by copying a service from counselling-services.json, or you can send the POST request that is already in route.rest
3. To check that the service has been successfully added, send the GET request at the top of the route.rest file (GET http://localhost:4000/counselling-services), which should return data that includes the service you added.
